From 70b1d2d1ea5f770e9747e1f428e32e45afa853a9 Mon Sep 17 00:00:00 2001
From: Fred Chen <fredchen.openbmc@gmail.com>
Date: Wed, 15 Oct 2025 20:33:52 +0800
Subject: [PATCH 1007/1008] ARM: dts: aspeed: santabarbara: Add eeprom device
 node for PRoT module

Add eeprom device node for PRot module FRU.

lore link: https://lore.kernel.org/all/20251015143916.1850450-8-fredchen.openbmc@gmail.com/

Signed-off-by: Fred Chen <fredchen.openbmc@gmail.com>
---
 .../arm/boot/dts/aspeed/aspeed-bmc-facebook-santabarbara.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-santabarbara.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-santabarbara.dts
index f0dd20233231..66d0d28e5216 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-santabarbara.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-santabarbara.dts
@@ -1076,6 +1076,11 @@ eeprom@50 {
 		reg = <0x50>;
 	};
 
+	eeprom@51 {
+		compatible = "atmel,24c128";
+		reg = <0x51>;
+	};
+
 	// BSM FRU
 	eeprom@56 {
 		compatible = "atmel,24c64";
-- 
2.49.0

